src/path/student_details.yaml

```diff
@@ -0,0 +1,103 @@
+post:
+  summary: Add the student details 
+  operationId: addStudentDetails
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: "../schemas/student_schemas/student_details_schema.yaml"
+  responses:
+    "201":
+      description: Student details have added successfully
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/common/response_message.yaml"
+    "500":
+      description: Internal server error
+      content:
+       application/json:
+          schema:
+            $ref: "../schemas/common/error_message.yaml"
+    "409":
+      description: The detail of the student already exists
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/common/error_message.yaml"
+get:
+  summary: Retrieve the student details based on ID
+  operationId: getSpecificStudentDetail
+  parameters:
+    -in: path
+    name: rollNo
+    required: true
+    description: The ID of the student to retrieve.
+    schema:
+      type: integer
+  responses:
+    "200":
```

Comment:
Here too. Is the response_message the expected response for getSpecificStudentDetail?